### DISCUSSION TITLE
1.7.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#911] [Обновления]
## Список изменений

- Добавлена возможность использования эксперементальной вариации голосов Яндекса. Новые голоса работают, только при некоторых условиях, во всех остальных случаях используются старые (#897)
- Добавлен обход кэша при получение перевода интерфейса после нажатия на кнопку "Обновить файлы локализации"
- Исправлена некорректная генерация Sec-{Vsubs|Vtrans}-Token заголовков ([vot.js#36](https://github.com/FOSWLY/vot.js/issues/36))
- Исправлена ошибка из-за которой старый плеер не добавлял lipsync эвенты после первого выключения перевода (#902, #894, возможно #887)
- Исправлена работа Yandex Disk по пути /i/ (#908)

## What's Changed
* update vot.js to 1.3.9 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/895
* Option to use an experimental variation of Yandex voices for some videos by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/898
* added cache bypass on force update localization files by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/901
* Updated chaimu to 1.0.3 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/903
* Fix yandex disk /i/ path by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/910

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.7.1...1.7.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.7.2'>1.7.2</a>.</em>

**Comment by MrDixioner** [comment]
Google Chrome 131.0.6778.69, Версия скрипта 1.7.2 - Failed to request create session.

**Reply by ilyhalight** [reply]
Ожидайте, проблемы у провайдера. Либо делайте по инструкции из #885

**Reply by FiLinX** [reply]
похоже что это уже веерно по провайдерам пошло.. у меня в МО та же ошибка на свежевышедшем видео

**Reply by ilyhalight** [reply]
уже вроде отпустило

**Reply by R0man1ch** [reply]
у меня всё ещё не работает в Safari 🥲
